lib/services/saml.go

```diff
@@ -49,18 +49,18 @@ func ValidateSAMLConnector(sc types.SAMLConnector, rg RoleGetter) error {
 	if sc.GetEntityDescriptorURL() != "" {
 		resp, err := http.Get(sc.GetEntityDescriptorURL())
 		if err != nil {
-			return trace.Wrap(err)
+			return trace.WrapWithMessage(err, "error getting entitydescriptor from %q for SAML connector:%v", sc.GetEntityDescriptorURL(), sc.GetName())
```

Comment:
entity descriptor is two words right?


Also, do we need to prefix the message with _error_? Won't it be pretty obvious that it's an error?

Reply:
Please see update @zmb3